invokeai/app/invocations/latent.py

```diff
@@ -211,7 +209,10 @@ def get_conditioning_data(self, model: StableDiffusionGeneratorPipeline) -> Cond
 
 
     def invoke(self, context: InvocationContext) -> LatentsOutput:
-        noise = context.services.latents.get(self.noise.latents_name)
+        if self.noise:
```

Comment:
I forgot to remove the `seed` parameter from this node, but we shouldn't need it, since it's not used elsewhere in the node. That would mean noise can't be optional, but I think that should be okay (since we'll be wrapping these lower-level nodes with shared graphs soon-ish).